### PR TITLE
Add response handling for non-json content types

### PR DESCRIPTION
**Pull request by tpharrison**
## Summary
Requests to Lighthouse will return responses with a content-type of HTML.

- Added parsing to handle responses for non-json content types.
- Refactoring - renamed some variables to be more consistent; reduced status-code data type conversions.
- Updated codeowners file.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/65093

## Testing done
- disability_compensations_controller_spec
- spec/requests/disability_compensation_form_request_spec.rb
- spec/lib/lighthouse/veteran_verification/service_spec.rb

### CODE REVIEW

**Review by jerekshoe — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -888,9 +888,13 @@ lib/lgy @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-af
 lib/lgy/configuration.rb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/lgy/service.rb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/lgy/tag_sentry.rb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-lib/lighthouse @department-of-veterans-affairs/lighthouse-banana-peels
+
+lib/lighthouse/service_exception.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/dbex-trex @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/lighthouse/benefit_claims @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (jerekshoe) [inline .github/CODEOWNERS:893]: Add dbex to this one

**Review by rmtolmach — APPROVED** [review]
CODEOWNER changes look good for the Platform perspective.

**Review by jerekshoe — DISMISSED** [review]

**Review by jerekshoe — DISMISSED** [review]

**Review by jerekshoe — APPROVED** [review]

**Review by jerekshoe — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
👌

**Review by mchae-nava — APPROVED** [review]
LGTM!

**Review by michelpmcdonald — APPROVED** [review]
approve, lighthouse-banana-peels team is no codeowners of direct deposit

**Pull request merged**